WSEvents.cpp

```diff
@@ -653,12 +656,36 @@ void WSEvents::StreamStatus() {
     obs_output_release(stream_output);
 }
 
+/************************************************************************************************************
+* Heatbeat is emitted every 2 seconds, when enabled with request:											*
+*                                        void WSRequestHandler::HandleSetHeartbeat(WSRequestHandler* req)	*
```

Comment:
Put the request type name instead of the function name. In that case it's `SetHeartbeat`.

Reply:
Done.